modules/serverless-knative-kafka-broker-metrics.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * serverless/observability/admin-metrics/serverless-admin-metrics.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-knative-kafka-broker-metrics_{context}"]
+= Knative Kafka broker metrics
+
+You can use the following metrics to debug and visualize the performance of Kafka broker.
+
+[cols=5*,options="header"]
+|===
+|Metric name
+|Description
+|Type
+|Tags
+|Unit
+
+|`event_count_1_total{job="kafka-broker-receiver-sm-service", namespace="knative-eventing"}`
+|Number of events received by a broker
+|Counter
+a|
+`name`:: broker name
+`namespace_name`:: broker namespace
+`event_type`:: event type
+`response_code`:: HTTP response code returned by the broker
+`response_code_class`:: HTTP response code class returned by the broker: 2xx, 3xx, 4xx, 5xx
+|Dimensionless
+
+|`event_dispatch_latencies_ms_bucket{job="kafka-broker-receiver-sm-service", namespace="knative-eventing"}`
+|The time spent dispatching an event to a Kafka cluster
+|Histogram
+a|
+`name`:: broker name
+`namespace_name`:: broker namespace
+`event_type`:: event type
+`response_code`:: HTTP response code returned by the broker
+`response_code_class`:: HTTP response code class returned by the broker: 2xx, 3xx, 4xx, 5xx
+|Milliseconds
+
+|`kafka_broker_controller_consumer_group_expected_replicas`
+|Number of expected replicas for a given Kafka consumer group resource
+|Gauge
+a|
+`consumer_name`:: resource name
+`namespace_name`:: resource namespace
+`consumer_kind`:: resource Kind, enum: `KafkaSource`, `Trigger`, `Subscription`
+
+[NOTE]
+====
+In this context, resources refers to user facing entities such as Kafka source, trigger, and subscription. Avoid using internal or generated names when using these resources.
```

Comment:
```suggestion
In this context, resources refer to user facing entities such as Kafka source, trigger, and subscription. Avoid using internal or generated names when using these resources.
```
For all other similar instances in the PR.

Reply:
Incorporated this throughout the content